packages/flame/src/Toaster/story.tsx

```diff
@@ -0,0 +1,177 @@
+import * as React from 'react';
+import { storiesOf } from '@storybook/react';
+import { withReadme } from 'storybook-readme';
+import Readme from './README.md';
+import { ToasterProvider, useToasts } from './index';
+import { ActionableToastContent, Toaster } from './Toaster';
+import { Button } from '../Button';
+import { Heading2 } from '../Text';
+
+const stories = storiesOf('Components|Toaster', module).addDecorator(withReadme(Readme));
+
+const noop = () => {};
+const Spacer: React.FC = ({ children }) => {
+  const nextChildren = React.Children.map(children, child => (
+    <div css={{ margin: '8px' }}>{child}</div>
+  ));
+  return <div css={{ display: 'flex' }}>{nextChildren}</div>;
+};
+
+const AutoDismissToastExample = () => {
+  const { addToast } = useToasts();
+
+  return (
+    <Spacer>
+      <Button
+        type="button"
+        onClick={() => {
+          addToast(`I'll disappear after 4 seconds`, {
+            appearance: 'success',
+          });
+        }}
+      >
+        Generate success
+      </Button>
+      <Button
+        type="button"
+        onClick={() => {
+          addToast(`I'll disappear after 4 seconds`, {
+            appearance: 'error',
+          });
+        }}
+      >
+        Generate error
+      </Button>
+    </Spacer>
+  );
+};
+
+const ActionableToastExample = () => {
+  const { addActionableToast } = useToasts();
+
+  return (
+    <Spacer>
+      <Button
+        type="button"
+        onClick={() => {
+          addActionableToast({
+            content: `I have an action!`,
+            actionTitle: 'Undo',
+            actionCallback: () => {
+              console.log('You clicked the undo button!');
+            },
+          });
+        }}
+      >
+        Generate success
+      </Button>
```

Comment:
Non-blocking: should we customize the default active/focus state for the close button or leave as is? This is with latest Chrome:

![image](https://user-images.githubusercontent.com/24464151/84070346-fe507e80-a999-11ea-8ccc-4f6373660ffe.png)

Reply:
@maartenafink